lib/Partitioner/Partitioner.cpp

```diff
@@ -695,6 +712,9 @@ void Partitioner::getBackendMap(
 
 llvm::Error Partitioner::createDAGWithoutPartition(
     BackendKind backendKind, std::map<BackendKind, BackendInfo> &backendMap) {
+  if (logPartition) {
+    llvm::outs() << "The model is too small for applying partition.\n";
```

Comment:
I'd like to get memory from device and memory required for a function dumped as well.
Would that be possible to get easily?